controller/productsController.js

```diff
@@ -0,0 +1,21 @@
+const productsService = require("../service/productsService");
+
+const getProducts = async (req, res, next) => {
+  const productCarousel = await productsService.getProducts();
+  const bigBannerUrl = await productsService.getBigBanners();
+  const smallBannerUrl = await productsService.getSmallBanners();
```

Comment:
변수 선언과 return문 사이에는 한 줄 띄워주세요~
```suggestion
  const smallBannerUrl = await productsService.getSmallBanners();
  
```